scarb/src/ops/new.rs

```diff
@@ -100,11 +102,17 @@ fn mk(
     // Create project directory in case we are called from `new` op.
     fsx::create_dir_all(&path)?;
 
-    init_vcs(&path, version_control)?;
-    write_vcs_ignore(&path, config, version_control)?;
+    let canonical_path = if let Ok(canonicalize_path) = fs::canonicalize(&path) {
```

Comment:
```suggestion
    let canonical_path = if let Ok(canonicalize_path) = fsx::canonicalize(&path) {
```

I think I would rather stick to our wrapper at all costs, because who knows if we wouldn't want to slightly adjust its global behaviour in the future (for example, use `dunce` on Windows).